src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -1126,22 +1124,22 @@ void process_mouse_over(const ScreenCoordsXY& screenCoords)
                     ScreenCoordsXY scrollCoords;
                     widget_scroll_get_part(
                         window, &window->widgets[widgetId], screenCoords, scrollCoords, &output_scroll_area, &scroll_id);
-                    cursorId = scroll_id;
+                    cursorId = static_cast<CursorID>(scroll_id);
```

Comment:
this line doesn't make any sense. Could you delete it. I think its just left over from the original.

Reply:
It does not compile if I remove the cast. scroll_id's type is different from CursorID, isn't it?

Reply:
I meant delete the whole line. Scroll id has absolutely nothing to do with cursor id.

Reply:
oh ok, got it.